### PR TITLE
Added mute video moderation feature

### PR DESCRIPTION
**Pull request by skolmer**
This change adds the ability for a moderator to deactivate the camera of other participants. The implementation uses the same approach that was implemented to mute audio of other participants.

This feature consists of multiple PRs:
- https://github.com/jitsi/jitsi-media-transform/pull/329
- https://github.com/jitsi/jitsi-videobridge/pull/1570
- https://github.com/jitsi/jitsi-xmpp-extensions/pull/37
- https://github.com/jitsi/jicofo/pull/695
- https://github.com/jitsi/lib-jitsi-meet/pull/1496
- https://github.com/jitsi/jitsi-meet/pull/8630

### CODE REVIEW

**Comment by damencho** [comment]
Could you please fix the lint errors, thank you.

**Comment by damencho** [comment]
Jenkins, test this please.

File: JitsiConferenceEventManager.js [hunk 1]
```diff
@@ -111,6 +111,30 @@ JitsiConferenceEventManager.prototype.setupChatRoomListeners = function() {
         }
     );
 
+    chatRoom.addListener(XMPPEvents.VIDEO_MUTED_BY_FOCUS,
+        actor => {
+            // TODO: Add a way to differentiate between commands which caused
+            // us to mute and those that did not change our state (i.e. we were
+            // already muted).
+            Statistics.sendAnalytics(createRemotelyMutedEvent());
```
Comment (damencho) [inline JitsiConferenceEventManager.js:119]: Hum, can we add a field to the analytics event whether this mute is audio or video?

**Review by damencho — CHANGES_REQUESTED** [review]
One small comment about changing the analytics, but other than that LGTM!

Good job, and thanks :)

File: JitsiConferenceEventManager.js (hunk 1 above)
Comment (skolmer) [reply JitsiConferenceEventManager.js]: Yeah, I just added the mediaType here too. 👍

**Comment by damencho** [comment]
Jenkins, test this please.

**Review by damencho — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**